pkg/actions/cluster/delete_test.go

```diff
@@ -0,0 +1,114 @@
+package cluster
```

Comment:
do we need the tests in this file? Is it not covered by the test in `unowned_test.go` and `owned_test.go`?

Reply:
I think that the naming of the file is not the best one.
In this file, I only test the `drainAllNodeGroups` function, while in `unowned_test.go` and `owned_test.go` I test the `Delete` function.
The reason this file is called `delete_test.go` is because that `drainAllNodeGroups` function is location in `delete.go` file

Reply:
gotcha :+1: